src/isolation/fdmap.rs

```diff
@@ -0,0 +1,71 @@
+use std::{
+	collections::HashSet,
+	os::fd::{FromRawFd, OwnedFd, RawFd},
+};
+
+#[derive(Default, Debug)]
+pub struct UhyveFileDescriptorMap {
+	fdset: HashSet<RawFd>,
+}
+
+impl UhyveFileDescriptorMap {
```

Comment:
This could be built into `UhyveFileMap`. It wasn't, so as to leave room for clarity and separate sandbox-related and fd-related functions.

Reply:
... as well as leave room for extensibility in the future. See https://github.com/hermit-os/uhyve/issues/897

Reply:
It's fine as is :-)